lib/view/crypto_list.dart

```diff
@@ -0,0 +1,160 @@
+import 'package:crypto_reminder/blocs/cryptocurrency_bloc.dart';
```

Comment:
I would also put this whole file in a sub-dir -> view/crypto or cryptocurrency or crypto_list

Reply:
Let's put this into `lib/view/crypto`